### ISSUE TITLE
Unmarshaler for `patternProperties` missing

### ISSUE THREAD

**Issue by sisp** [grktsh/python-oas#6]
I can't find an unmarshaler for `patternProperties`.

**Comment by sisp** [comment]
`patternProperties` is not supported by OpenAPI 3.0: https://github.com/OAI/OpenAPI-Specification/issues/687#issuecomment-287806750

In this case, the `patternProperties` validator must raise an error.

**Comment by grktsh** [comment]
> Even though python-oas can validate patternProperties due to jsonschema, it is not officially supported and data is not unmarshaled based on patternProperties anyway.

Thanks, I overlooked this behavior.  I'd like to remove `patternProperties` validator from `oas.schema.validators._Validator` since it is unsupported feature and it is incomplete behavior that data is validated but is not unmarshaled. What do you think?

**Comment by sisp** [comment]
I agree that it would be more consistent to not validate `patternProperties`.

**Comment by sisp** [comment]
How would you go about removing the validator? `jsonschema` only seems to have the `extend` method. After checking the `jsonschema` source code, I think simply removing the validator from `_Validator.VALIDATORS` should be safe as `_Validator` is a custom extension.

```python
_Validator = validators.extend(
    Draft4Validator, {'type': _type_validator, 'enum': _enum_validator}
)

for name in [u'patternProperties']:
    _Validator.VALIDATORS.pop(name)
```

**Comment by grktsh** [comment]
Agree. But I think del statement is better than dict.pop since the return value is not used. And I'd like to avoid polluting the namespace with `name`.  How about `del _Validator.VALIDATORS['patternProperties']`?

**Comment by sisp** [comment]
LGTM. I can create a PR if you like.

**Comment by grktsh** [comment]
Could you create it? 🙏

**Comment by sisp** [comment]
I've created a PR. Let me know what you think.
